### PR TITLE
Enable v8 by default

### PR DESCRIPTION
**Pull request by aboodman**
Third attempt at landing this. Now we enable it by default except with an env var that confirms.

### CODE REVIEW

File: go/constants/version.go [hunk 1]
```diff
@@ -5,7 +5,29 @@
...
 package constants
 
+import (
+	"fmt"
+	"os"
+)
+
 // TODO: generate this from some central thing with go generate, so that JS and Go can be easily kept in sync
-const NomsVersion = "7"
+const NomsVersion = "7.1"
```
Comment (aboodman) [inline go/constants/version.go]: I decided to do this so that next stable version could be `8`.

**Comment by arv** [comment]
The failure looks it is due to this PR

File: go/constants/version.go (hunk 1 above)
Comment (arv) [inline go/constants/version.go]: Seems OK.

**Comment by aboodman** [comment]
What MD files? I updated the ones I could think of.

**Comment by aboodman** [comment]
jenkins: test this

**Comment by aboodman** [comment]
jenkins: test this

**Comment by aboodman** [comment]
@arv tests pass now.

**Comment by arv** [comment]
> What MD files? I updated the ones I could think of.

I meant that you could include the changes to the docs in this PR but if you want to do that in a different PR that is fine with me.

**Comment by aboodman** [comment]
Oh I actually changed the md files already using the github UI. Let me know
if you see one that still needs fixing.

**Pull request merged**